bndtools.core/bndtools.shared.bndrun

```diff
@@ -63,6 +104,7 @@ crossplatform.bundles: \
 	org.osgi.impl.bundle.repoindex.lib,\
 	${help.bundles},\
 	${p2.bundles},\
+	${m2e.bundles},\
```

Comment:
So I have drastically simplified this now.  but I still feel we should just add the m2e bundles to the main `bndtools.bndrun` test because 1 ) it doesn't require m2e to be installed in parent/host eclipse, 2) it is too much work to maintain a separate group of top level bundruns unless we could do some type of parameterized includes.